SalesforceCommerceCloud/pwa-kit
# Upgrade `copy-webpack-plugin` to fix `tar` vulnerability.

# Description

This small PR does 2 things, upgrades `copy-webpack-plugin` to resolve the `tar` vulnerability (it actually removes the tar dependency all together for that package) and re-locks all the projects lock files so dependencies are up-to-date.


NOTE: We've update to the latest minor version of @chakra-ui/react as they introduced a bug causing jest tests to fail.



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Upgrade `copy-webpack-plugin`

# How to Test-Drive This PR

- Checkout branch
- Run `npm ci` 
- In the SDK project folder run `npm ls tar`
- There should be 0 results

# Checklists




## General
